lib/easy.c

```diff
@@ -95,6 +95,11 @@ static void win32_cleanup(void)
 #endif
 }
 
+#ifdef WIN32
```

Comment:
this should be _WIN32 imo (that's standard check for Windows build)

Reply:
We use plain `WIN32` for this in multiple places already. See:

https://github.com/curl/curl/blob/0299b262cd9c75adab546f4851c03995d98d61e1/lib/curl_setup.h#L35